File: WordCounter.vb
Module Module1

    Sub Main()
        Dim myStr, myChar As String
        Dim Counter, wordCount As Integer

        myStr = ""
        myChar = ""
        Counter = 0
        wordCount = 0

        Console.Write("Enter string to count words: ")
        myStr = Console.ReadLine

        For Counter = 1 To Len(myStr)
            myChar = Mid(myStr, Counter, 1)
            If myChar = " " Then
                wordCount = wordCount + 1
            End If
        Next

        If Right(myStr, 1) <> " " Then
            wordCount = wordCount + 1
        End If

        Console.WriteLine("The word count in sentense """ & myStr & """ is = " & wordCount)
        Console.ReadKey()

    End Sub

End Module

File: WordCounter/wordCounter.py
wordCount = 0

myStr = input("Enter string to count words: ")

for Counter in range(len(myStr)):
    myChar = myStr[Counter:Counter+1]
    if myChar == " ":
        wordCount = wordCount + 1

if myStr[-1:] != " ":
    wordCount = wordCount + 1

print("The word count in sentence \"", myStr, "\" is =", wordCount)